taxi/forms.py

```diff
@@ -0,0 +1,57 @@
+from django import forms
+from django.contrib.auth import get_user_model
+from django.contrib.auth.forms import UserCreationForm
+from django.core.exceptions import ValidationError
+
+from taxi.models import Driver, Car
+
+
+class CarForm(forms.ModelForm):
```

Comment:
check [checklist](https://github.com/mate-academy/py-taxi-service-user-form-and-validation/blob/master/checklist.md)